test cases/failing build/3 pch disabled/test_args.txt

```diff
@@ -0,0 +1,6 @@
+# This file is not read by meson itself, but by the test framework.
+# It is not possible to pass arguments to meson from a file.
+# Disable PCH usage to make sure backends respect this setting.
+# Since the .c file requires PCH usage (it does not include necessary
+# headers itself), the build should fail.
+['-D', 'b_pch=false']
```

Comment:
Shouldn't this use a `default_options` declaration in the `project()` call instead?

Reply:
Don't they behave the same for the test? I copied this from an old test, maybe this was from before `default_options` was added.

Reply:
Changed to use `default_options`.